projects/views.py

```diff
@@ -1,6 +1,36 @@
 from django.shortcuts import render
+from .models import Post
+import random
 
 # Create your views here.
 def home(request):
-    return render(request, 'home.html')
+    posts = Post.objects
+    return render(request, 'home.html', {'posts': posts})
+
+
+def post_detail_view(request):
+    print(request)
+    posts = Post.objects
+    template = "post_detail_view.html"
+    context = {}
+    return render(request, template, {'posts': posts}, context)
+
+
+def room86(request):
+    print(request)
+    random_posts(request)
```

Comment:
1. add a query to get all posts  here
2. get one post and using random 
3. add both variables in the context